### PR TITLE
Silent logging

### PR DESCRIPTION
**Pull request by dtpryce**
- Removed the v option from tar
- Sent output of activate to dev/null 

Was hard to test with the mlserver image being built elsewhere

### CODE REVIEW

File: hack/activate-env.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ _activateEnv() {
   echo "--> Sourcing new environment at $_envFolder..."
   # Need to disable unbound errors for activate
   set +u
-  source $_activate
+  source $_activate &> /dev/null 2>&1
```
Comment (adriangonz) [inline hack/activate-env.sh:39]: I would keep the error output visible - in case it fails to activate the environment somehow

**Review by adriangonz — COMMENTED** [review]
Nice one @dtpryce! Added a small comment there, but the approach make sense :+1:

File: hack/activate-env.sh (hunk 1 above)
Comment (dtpryce) [reply hack/activate-env.sh]: Updated to be only `1> /dev/null` so that should silence all non-error logging

**Review by adriangonz — APPROVED** [review]

**Comment by dtpryce** [comment]
Awesome @adriangonz feel free to merge into 1.4.0 or earlier :P

**Comment by dtpryce** [comment]
> Awesome @adriangonz feel free to merge into 1.4.0 or earlier :P

Actually wait I have another small update for it

**Comment by adriangonz** [comment]
Nice one! This should be ready to go. Thanks for the changes @dtpryce ! :rocket:

**Pull request merged**